parameter_files/fates_params_default.cdl

```diff
@@ -264,6 +301,18 @@ variables:
 	double fates_mort_bmort(fates_pft) ;
 		fates_mort_bmort:units = "1/yr" ;
 		fates_mort_bmort:long_name = "background mortality rate" ;
+	double fates_mort_ip_senescence(fates_pft);
```

Comment:
@rgknox @JessicaNeedham for clarity/readability sake, can we add "_size_" to these two parameter names to better distinguish from the age ones?

Reply:
Yes, that's a good idea.